<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compensation System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>Compensation System</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 18px; font-weight: 600;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="loginGroup">
      <property name="title">
       <string>Login</string>
      </property>
      <layout class="QGridLayout" name="loginGridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="roleLabel">
         <property name="text">
          <string>Role</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="roleCombo">
         <item>
          <property name="text">
           <string>CITIZEN</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>OFFICER</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="usernameLabel">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="usernameEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="passwordLabel">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="passwordEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="loginButton">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="logoutButton">
         <property name="text">
          <string>Logout</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="sessionLayout">
      <item>
       <widget class="QLabel" name="sessionLabel">
        <property name="text">
         <string>Not logged in</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="contentStack">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="submitPage">
       <layout class="QVBoxLayout" name="submitPageLayout">
        <item>
         <widget class="QGroupBox" name="submitGroup">
          <property name="title">
           <string>Submit Claim (Citizen)</string>
          </property>
          <layout class="QHBoxLayout" name="submitHorizontalLayout">
           <item>
            <widget class="QLabel" name="claimIdLabel">
             <property name="text">
              <string>Claim ID (8 digits)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="claimIdEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="submitButton">
             <property name="text">
              <string>Submit + Calculate</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="viewListButton">
          <property name="text">
           <string>Go To Claim List</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="submitVerticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="listPage">
       <layout class="QVBoxLayout" name="listPageLayout">
        <item>
         <layout class="QHBoxLayout" name="listActionsLayout">
          <item>
           <widget class="QPushButton" name="refreshButton">
            <property name="text">
             <string>Refresh Claims</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="backToSubmitButton">
            <property name="text">
             <string>Go To Submit Form</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="claimsTable">
          <property name="columnCount">
           <number>8</number>
          </property>
          <column>
           <property name="text">
            <string>Claim ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Claimant ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Income</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Compensation</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Submitted</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
